### ISSUE TITLE
Allow Multiple Logins on One Game Provider

### ISSUE THREAD

**Issue by sunseraphic** [gogcom/galaxy-integrations-python-api#149]
For each of the game providers, one player may have multiple accounts and each account may own different games. Unfortunately, now GOG Galaxy only allow only one login on each game provider.

Is it possible to support multiple logins for any of the game providers? It would empower GOG users to manage all owned game with the one App.

**Comment by r6d2** [comment]
I second this. I have several Steam accounts for all family members and would like to see a simple way to integrate them on s single view to avoid buying the same game more than once.

**Comment by dreammaster38** [comment]
Yes this is a feature i would like to see too. We have also different family accounts on different providers (Steam, Uplay). Thios would be a really great feature!

**Comment by Keristero** [comment]
Yup, I have two steam accounts.

**Comment by Spedwards** [comment]
I have two Steam accounts, two uPlay accounts, and three Battle.net accounts. Would love to be able to add all of them.

**Comment by Firehawke** [comment]
I have multiple XBL and PSN accounts for different regions, so this would be very helpful.

**Comment by maltesserver** [comment]
This is still not addressed....

**Comment by hlc1209** [comment]
+1

**Comment by babsevensix** [comment]
Hi, i have multiple account on origin... Some news about this task?

**Comment by LittleFreak** [comment]
Cannt you just copy/rename the Provider?

**Comment by AlessiOrl** [comment]
I second this, i have multiple accounts in different platforms and i would like to have them

**Comment by McBruno712** [comment]
I have 2 Steam accounts and my brother (whom I share my PC with) has another, so this feature would be very much appreciated!

**Comment by ChickenDestroyer** [comment]
I too would like this.
